macros/supporting/ghost_record_per_datatype.sql

```diff
@@ -96,6 +102,7 @@
 {%- elif ghost_record_type == 'error' -%}
 
         {%- if datatype == 'TIMESTAMP' or datatype == 'TIMESTAMP WITH LOCAL TIME ZONE' %} {{- datavault4dbt.string_to_timestamp( timestamp_format , end_of_all_times) }} as "{{ column_name }}"
+        {%- elif datatype == 'DATE'-%} TO_DATE('{{ end_of_all_times_date }}', '{{ date_format }}' ) as "{{ alias }}"
```

Comment:
@tta-scalefree @tkirschke This leads to an error in BigQuery because BigQuery does not allow column names with quotes